packages/kotti-ui/source/kotti-comment/KtComment.vue

```diff
@@ -157,3 +157,10 @@ export default {
 	},
 }
 </script>
+
+<style lang="scss" scoped>
+.action__reply {
```

Comment:
as with the class restructuring i'd propose, you also look into how you can make this follow the BEM convention, so it's not `action__reply` but whatever it is in the hierarchy of `kt-comment`